src/Kunstmaan/NodeBundle/Tests/Entity/NodeTest.php

```diff
@@ -161,6 +170,36 @@ public function testToString()
         $this->assertEquals('node 1, refEntityName: Kunstmaan\NodeBundle\Tests\Entity\TestEntity', $this->object->__toString());
     }
 
+
+	public function testPageShouldHaveTab() {
+		$request = new Request();
+		$request->request = new ParameterBag();
+
+		$formFactory = $this->getMock(FormFactory::class, [], [], '', false);
+		$authorizationChecker = $this->getMock(AuthorizationCheckerInterface::class);
+		$permissionAdmin = $this->getMock(PermissionAdmin::class, [], [], '', false);
+		$permissionMap = $this->getMock(PermissionMapInterface::class);
+
+		$entity = new TestEntity();
+		$this->object->setRef($entity);
+
+		$tabPane = new TabPane('id', $request, $formFactory);
+
+		$adaptFormEvent = new AdaptFormEvent($request, $tabPane, $entity);
+
+		$nodeListener = new NodeListener($authorizationChecker, $permissionAdmin, $permissionMap);
```

Comment:
as mentioned in my other comment, none of those dependencies are actually needed to test your behaviour, they are just noise.